main.js

```diff
@@ -1,74 +1,78 @@
+/* eslint-disable max-classes-per-file */
+/* eslint-disable no-use-before-define */
+/* eslint-disable no-unused-vars */
 const displaySection = document.querySelector('.book_list');
 const addBtn = document.querySelector('#add_btn');
 const title = document.querySelector('#title');
 const author = document.querySelector('#author');
 
-let availableBooks = [];
+class classAddBooks {
+  constructor() {
+    this.availableBooks = [];
+  }
+
+  static addBook = (element) => {
+    element.preventDefault();
+    const addedBook = {
+      title: title.value,
+      author: author.value,
+    };
+
+    this.availableBooks.push(addedBook);
+    clear();
+    ClassLocalStorage.saveToLocalStorage(this.availableBooks);
+    displayItem();
+  };
```

Comment:
- [ ] Great job so far, when I try to add books it's giving me this error, kindly fix that.

![Screen Shot 2022-09-22 at 9 14 02 AM](https://user-images.githubusercontent.com/59416136/191671747-7122e9d8-c153-45cb-83e9-66f269dbd2ad.png)

Reply:
hello, thanks for the review. 
it actually works fine for me with no error.

Reply:
please recheck the project. I think the error might be on your part, as it works well for me. have rerun the project several times.